CHANGELOG.md

```diff
@@ -108,6 +108,20 @@ ServiceDescription("First", "Second", "DontCare") myServiceDescription2;
 ServiceDescription("Foo", "Bar", "Baz") myServiceDescription3;
 ```
 
+The service-related methods have been move from `PoshRuntime` to a separate class (TBD):
```

Comment:
We should add a checkbox to our PR template: `Check if public API changes are added to CHANGELOG.md` for the reviewer.

Later this can be very helpful. Thanks for adding it directly.

Reply:
This is already in the current PR template. However, it was added after this PR was created. Added it manually.